client/my-sites/plans/plans-features/google-analytics.js

```diff
@@ -60,72 +63,44 @@ export default function( context ) {
 				</span>
 				<div className="feature-analytics__ilustration" dangerouslySetInnerHTML={{ __html: svgIllustration }} />
 			</Card>
-			{ site && site.plan && site.plan.product_name_short === 'Business'
+			{ planSlug === 'business-bundle'
 				? <Card href={ '/settings/analytics/' + site.slug }>{ i18n.translate( 'Configure Google Analytics' ) }</Card>
 				: <Card compact className="feature-analytics__plan-compare">
```

Comment:
I'm thinking about abstracting all these CSS classes for reuse. What do you think?